### PR TITLE
add citation_text_converter

### PR DESCRIPTION
**Pull request by sohamsshah**
# Description

Add the Citation Text Converter script to this project. This Script converts the BibTex/DOI etc. formats to the Citation Text format and helps the Researchers to relieve from the menace of manually writing them. More information in the documented README.md file in the project. Dependancies: [citation-js](https://www.npmjs.com/package/citation-js) npm package. 
Language: JavaScript
Runtime Environment: Node.js

Fixes in ISSUE NUMBER: #422 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# Checklist:

- [X] My code follows the style guidelines(Clean Code) of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sohamsshah** [comment]
@HarshCasper Please check the Script out. Also I am having checks fails, if that is significant; then do guide too to resolve it! ISSUE #422

File: JavaScript/Citation_Text_Converter/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+## REFERENCES:
+* [Official Documentation](https://citation.js.org/api/index.html)
+* Download the npm package: [citation-js](https://www.npmjs.com/package/citation-js)
+
+---
+
+Made with 😎 by [Soham Shah](https://github.com/sohamsshah) (codeFlayer)
+
```
Comment (HarshCasper) [inline JavaScript/Citation_Text_Converter/README.md:60]: Remove these extra lines in README

File: JavaScript/Citation_Text_Converter/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+## INTRODUCTION:
+
+Citation Text converter can easily convert your DOI, BibTeX and other well known formats to the Citation Text, just by integrating this script. And not just only Citation Text, but this also supports other conversions such as DOI to BibTeX, BibTeX to DOI etc.
+
+---
+
+## GET STARTED:
```
Comment (HarshCasper) [inline JavaScript/Citation_Text_Converter/README.md:25]: You can remove these parts and just add the commands of running the script via NPM

File: JavaScript/Citation_Text_Converter/README.md (hunk 1 above)
Comment (sohamsshah) [reply JavaScript/Citation_Text_Converter/README.md:60]: Thank you for such a nice review buddy!

File: JavaScript/Citation_Text_Converter/README.md (hunk 2 above)
Comment (sohamsshah) [reply JavaScript/Citation_Text_Converter/README.md:25]: Ohkay!

**Comment by sohamsshah** [comment]
@HarshCasper Changed and commited according to the Review. Please have a look.

Changes Summary:
* Removed extra lines (in the README.md and the code)
* Add Comments in the code that depicts the Output 
* Add a step about how to start the script (as node index.js in README.md)

Thank you.

**Comment by sohamsshah** [comment]
@HarshCasper Hey Man, Can you please review?

**Comment by HarshCasper** [comment]
@all-contributors bot please add @sohamsshah for code

**Comment by HarshCasper** [comment]
@all-contributors bot please add @sohamsshah for documentation

**Pull request merged**